src/main/scala/esmeta/analyzer/util/Stringifier.scala

```diff
@@ -138,6 +142,8 @@ class Stringifier(
       case UncheckedAbruptCompletionMismatch(riap, actual) =>
         app >> "[UncheckedAbruptCompletionMismatch] " >> riap
         app :> "- actual  : " >> actual
+      case AssertionMismatch(asp) =>
+        app >> "[AssertionMismatch] " >> asp >> " failed"
```

Comment:
```suggestion
        app >> "[AssertionMismatch] " >> asp
```
nit: I think a `...Mismatch` prefix already represents it is failed

Reply:
I thought it was uncertain because the name was `AssertionMismatch`, not `AssertionFailed`. It seems better to change the word itself rather than correct it. Maybe require wide change(not restricted to this error, but many other PRs)